packages/appChrome/components/AppChrome.tsx

```diff
@@ -9,54 +9,49 @@ import {
 } from "../../shared/styles/styleUtils";
 import { ThemeProvider } from "@emotion/react";
 import getCSSVarValue from "../../utilities/getCSSVarValue";
-import { themeBgPrimaryInverted } from "../../design-tokens/build/js/designTokens";
+import { themeBgPrimary } from "../../design-tokens/build/js/designTokens";
 import {
   defaultSidebarItemHorizPaddingSize,
   defaultSidebarItemVertPaddingSize
 } from "./SidebarItem";
+import { defaultHeaderPaddingHor, defaultHeaderPaddingVert } from "./HeaderBar";
 
 export interface AppChromeProps {
   sidebar: React.ReactNode;
   headerBar: React.ReactNode;
   mainContent: React.ReactNode;
 }
 
-class AppChrome extends React.PureComponent<AppChromeProps, {}> {
-  public render() {
-    const { sidebar, headerBar, mainContent } = this.props;
-
-    return (
-      <ThemeProvider
-        theme={{
-          sidebarBackgroundColor: getCSSVarValue(themeBgPrimaryInverted),
-          sidebarItemPaddingHor: defaultSidebarItemHorizPaddingSize,
-          sidebarItemPaddingVert: defaultSidebarItemVertPaddingSize
-        }}
+const AppChrome = ({ sidebar, headerBar, mainContent }: AppChromeProps) => {
+  return (
+    <ThemeProvider
+      theme={{
+        sidebarBackgroundColor: getCSSVarValue(themeBgPrimary),
+        sidebarItemPaddingHor: defaultSidebarItemHorizPaddingSize,
+        sidebarItemPaddingVert: defaultSidebarItemVertPaddingSize,
+        headerPaddingHor: defaultHeaderPaddingHor,
+        headerPaddingVert: defaultHeaderPaddingVert
+      }}
+    >
+      <div
+        className={cx(appChrome, textSize("m"), flex({ direction: "column" }))}
+        data-cy="appChrome"
       >
-        <div
-          className={cx(
-            appChrome,
-            textSize("m"),
-            flex({ direction: "column" })
-          )}
-          data-cy="appChrome"
-        >
-          <div data-cy="headerBar">{headerBar}</div>
-          <div className={cx(flex(), appWrapper)}>
-            <div className={flexItem("shrink")} data-cy="sidebar">
-              {sidebar}
-            </div>
-            <main
-              className={cx(flexItem("grow"), flush("left"), appWrapper)}
-              data-cy="main"
-            >
-              {mainContent}
-            </main>
+        <div data-cy="headerBar">{headerBar}</div>
+        <div className={cx(flex(), appWrapper)}>
+          <div className={flexItem("shrink")} data-cy="sidebar">
+            {sidebar}
           </div>
+          <main
+            className={cx(flexItem("grow"), flush("left"), appWrapper)}
+            data-cy="main"
+          >
+            {mainContent}
+          </main>
         </div>
-      </ThemeProvider>
-    );
-  }
-}
+      </div>
+    </ThemeProvider>
+  );
+};
 
-export default AppChrome;
+export default React.memo(AppChrome);
```

Comment:
Is it a breaking change though? I asked myself this. I am changing a fallback value but these values seem to reference DC/OS color variables, and that value is only used for AppChrome, so the fallback value isn't even being used at all with the override in the UI. 
The style.ts file handles this sort of logic, for example, this is how it handles the background color:
`  const bgColor =
    theme.headerBackgroundColor || getCSSVarValue(themeBgAppHeader);`

Reply:
I will leave the theme to take care of this as we already have the specific theming variable. There will be more design token changes to come in the near future with the audit.

Reply:
With the future design token changes will these also be breaking changes? If so, would it make sense to do them all at once?

Reply:
This one directly relates to the AppChrome component so I would like to change it here ideally. I think this will help with the issues we've seen trying to use npm link. I saw these issues when updating Emotion in the UI as well. This color is only used with this component.